app/controllers/issues_controller.rb

```diff
@@ -37,4 +41,18 @@ def issues_params
                                   :category_id, :description, :attachment,
                                   :location, :title)
   end
+
+  def find_relevant(category_id, issue_id)
+    Issue.where(category_id: category_id)
+         .where.not(id: issue_id)
+  end
+
+  def issue_status(status)
+    case status
```

Comment:
this is must be a helper class.
AND IT WAS IMPLEMENTED IN SOME PR (ON ISSUES LIST)

WORK TOGHETHER AS A TEAM